src/components/templates/Jazzbar/JazzTab/constants.ts

```diff
@@ -1,5 +1,10 @@
+import { DEFAULT_TABLE_CAPACITY } from "settings";
+
 import { Table } from "types/Table";
 
 import { generateTables } from "utils/table";
 
-export const JAZZBAR_TABLES: Table[] = generateTables({ num: 12, capacity: 6 });
+export const JAZZBAR_TABLES: Table[] = generateTables({
+  num: 12,
```

Comment:
This should probably also be a `const` in `src/settings`

Reply:
fixed in 628728c978f8946518261612288a3324163ca176